docs/iris/example_code/Meteorology/COP_1d_plot.py

```diff
@@ -81,8 +81,7 @@ def main():
                 label='pre-industrial', lw=1.5)
 
     # Establish where r and t have the same data, i.e. the observations
-    common = np.where(a1b_mean.data == e1_mean.data)[0]
-    observed = a1b_mean[common]
+    observed = a1b_mean[:np.argmin(np.isclose(a1b_mean.data, e1_mean.data))]
```

Comment:
The `np.argmin` version *is* kinda hard to unpick. In part because it does so much in one line. How about?
```python
close = np.isclose(a1b_mean.data, e1_mean.data)
first_different = np.where(~close)[0]
observed = a1b_mean[:first_different]
```